mailgun/exceptions.py

```diff
@@ -0,0 +1,3 @@
+class HttpResponseException(RuntimeError):
```

Comment:
The intent is for it to encapsulate any `django.http.HttpResponse` subclass (which I should probably be asserting on init, actually), hence the name.  It's there largely because I miss flask's `abort()` method (okay, and because my understanding of `raise` was wrong, it won't let me raise an instance of a new-style object that doesn't derive from `Exception`).  There doesn't seem to be a builtin way in django for you to just short-circuit and return an HttpResponse from deep in the call stack in your view.  Which, given that that's the point where you know specifically the type and details of the response you want, seems silly.  

An alternative design is to create a collection of HttpException objects, subclassed per status code, and catch *that* in the handler, construct the response, and return it.  That is probably slightly cleaner (in that the thing throwing the error doesn't have to know the format of the response you're expected to return), but it felt like overkill.  Let me know if you strongly prefer that approach.